### ISSUE TITLE
Test Elections for accessibility

### ISSUE THREAD

**Issue by msayoung** [localgovdrupal/localgov_elections#177]

**Comment by msayoung** [comment]
From Will https://localgovdrupal.slack.com/team/USF2TDXV1
The results graph needs an accessibility check (flagging to [@Maria (Agile Collective)](https://localgovdrupal.slack.com/team/U01C4FXU0F7) and the accessibility group)

**Comment by willguv** [comment]
Thanks @msayoung for writing an issue. I should have just written one!

**Comment by msayoung** [comment]
It already existed, I just reminded myself of it :)

**Comment by tonypaulbarker** [comment]
Moved to drupal.org https://www.drupal.org/project/localgov_elections/issues/3572818
